### ISSUE TITLE
Issue installing Apeman

### ISSUE THREAD

**Issue by baty0man** [hotnops/apeman#36]
Hi,

Running the following command

`python -m init.aws_initialize -o ../import`

I'm getting the following error:
`[*] Writing new schema to awschema.json
[*] Service name: awsre_postprivate not found
[*] Service name: awsre_postprivate not found
[*] Failed to connect retrying in 5 seconds
[*] Failed to connect retrying in 5 seconds
[*] Failed to connect retrying in 5 seconds
[*] Failed to connect retrying in 5 seconds
[*] Failed to connect retrying in 5 seconds
[*] Failed to connect retrying in 5 seconds
[*] Failed to connect retrying in 5 seconds
[*] Failed to connect retrying in 5 seconds`

Would you know what is the issue? Cheers

**Comment by hotnops** [comment]
Hmm. The service name not found is normal. Is your neo4j container running? That failed to connect message is the attempt to connect to the neo4j database for ingest.

**Comment by baty0man** [comment]
Yep the issue was coming from the neo4j, the configuration in the docker compose file was using too much memory so I decreased it and it worked.

Now, I'm getting this issue when running the analyze.analyze

```
venv) yo@yo-ubuntu-vm:~/Git/apeman/utils$ python -m analyze.analyze
[*] Populating ARN fields
[*] Expanding resources types
[*] Expanding action blobs
[*] Expanding resource blobs
[*] Expanding principal blobs
[*] Analyzing assume roles
[!] Could not analyze assume roles
--Return--
> /home/yo/Git/apeman/utils/analyze/analyze.py(124)analyze_assume_roles()->None
-> import pdb; pdb.set_trace()
(Pdb)
```

**Comment by hotnops** [comment]
It sounds like there may be some networking issues with the docker container, if it can't reach http://apeman-backend.localhost. What version of docker are you using? Anything different with your ip tables?
